content/docs/getting-started.md

```diff
@@ -9,17 +9,21 @@ weight: 10
 
 ## Building the dex binary
 
-Dex requires a Go installation and a GOPATH configured. For setting up a Go workspace, refer to the [official documentation][go-setup]. Clone it down the correct place, and simply type `make` to compile the dex binary.
+To build dex from source code, install a working Go environment with version 1.15 or greater according to the [official documentation][go-setup].
+Then clone the repository down to the correct place, and simply type `make` to compile the dex binary.
```

Comment:
I'm not sure where the "correct place" is, and the example that follows provides a path, so we may not need to mention location here.

```suggestion
Then clone the repository, and type `make` to compile the dex binary.
```

Reply:
Good improvement, thanks.